stage0/run.go

```diff
@@ -337,6 +337,18 @@ func Run(cfg RunConfig, dir string) {
 		}
 	}
 
+	destRootfs := common.Stage1RootfsPath(dir)
+	flavor, err := os.Readlink(filepath.Join(destRootfs, "flavor"))
+	if err != nil {
```

Comment:
3rd party stage1 images will bail out here. We need to handle the lack of the symlink gracefully.